packages/wrangler/src/index.tsx

```diff
@@ -467,6 +469,38 @@ export function createCLIParser(argv: string[]) {
 		}
 	);
 
+	// type generation
+	wrangler.command(
+		"types",
+		"📝  Generate types from bindings & module rules in config",
+		() => {},
+		async () => {
+			await printWranglerBanner();
+			const config = readConfig(undefined, {});
+
+			const configBindings: PartialConfigToDTS = {
+				kv_namespaces: config.kv_namespaces || [],
```

Comment:
> Also @JacobMGEvans the force pushing means it's impossible to use the "changes since last review" view on GitHub (unless there's some workaround for that)

You technically can look at the commit's between the forces, it usually diffs them.

Reply:
We should look at changing it from the code I pulled this from too then:
https://github.com/cloudflare/wrangler2/blob/965ab19b6a2fca6ef16928baadc3436677d78c80/packages/wrangler/src/publish/publish.ts#L482-L505